Azure/sonic-buildimage
# [Mellanox] Add support for SN4600 system

**- Why I did it**
Add support for new 64x200G SN4600 systems

**- How I did it**
Added all relevant files (w/o platform.json and hwsku.json as they will come later) with default SKU.

**- How to verify it**
Install image on switch, verify all ports are up and configured properly, run full platform SONiC tests.

**- Which release branch to backport (provide reason below if selected)**




**- Description for the changelog**



**- A picture of a cute animal (not mandatory but encouraged)**
